build.sh

```diff
@@ -170,60 +164,20 @@ function docker_build_with_version {
 
   pull_image "$dockerfile"
 
+  docker info 2>/dev/null | grep podman 1>/dev/null
+  is_podman=$?
+
+  if [[ "$SKIP_SQUASH" == 0 ]] && [[ $is_podman -eq 0 ]]; then
```

Comment:
One condition uses `==` and the second one uses `-eq`, not a real problem, just a nitpick.